content/_data/changelogs/weekly.yml

```
@@ -5223,6 +5223,74 @@
   # pull: 4303 too minor (PR title: Use actual logger to record Jenkins home and where it was found)
   # pull: 4313 (PR title: [JENKINS-58625] Cancel older builds of a given branch)
 
+- version: '2.203'
+  date: 2019-11-05
+  changes:
+
+# Major RFE
+  - type: major rfe
+    issue: 19887
+    message: |-
+      Allow time zone to be set on a per user basis
+    pull: 4113
+    authors:
+      - silent-snowman
+
+  # RFEs
+
+  - type: rfe
+    message: |-
+      Minor improvements to the logging UI
```

Too minor

---

Alternatively, be more specific, e.g. 

```suggestion
      Logging UI: Reorder sidepanel entries, add a note that "all log messages" will only include entries on level NOTE and up.
```